### PR TITLE
feat(feedback): add some feedback for the users

### LINKED ISSUE #5: Show information about compilation (started, progress etc)

### PR DESCRIPTION
**Pull request by robertohuertasm**
Partially fixes #5 

It shows some messages for the user to get some feedback. It also shows the `stadout` which was not used in the previous version and has some nice information about some parameters missing in the default `Cargo.toml`.

@xtuc I introduced a new dependency to get some emojis in the messages. I thought that it was nice to follow the same approach as `wasm-pack` when dealing with the feedbak but it's totally up to you if that's ok or not. Let me know if you don't like it and I will remove it. 😉

### CODE REVIEW

File: loader.js [hunk 1]
```diff
@@ -1,26 +1,29 @@
...
+const log = require('console-emoji');
 
 function spawnWasmPack({ isDebug, cwd }) {
   const bin = 'wasm-pack';
 
   const args = [
     '--verbose',
     'build',
-    '--target', 'browser',
-    '--mode', 'no-install',
```
Comment (xtuc) [inline loader.js]: I liked that formating, but I looks like you prettified it. That's fine too.

File: loader.js [hunk 2]
```diff
@@ -54,19 +57,25 @@ module.exports = function() {
   const cwd = dirname(this.resourcePath);
   const pkgDir = join(cwd, './pkg');
 
+  log(`:information_source:  Compiling your Rust code...`, 'green');
```
Comment (xtuc) [inline loader.js]: Instead of rust code or library I think that `crate` is the correct term (Cargo.toml is its manifest).

File: loader.js (hunk 1 above)
Comment (robertohuertasm) [reply loader.js]: Yes, sorry. I have prettier by default in my IDE. I can deactivate it if you want. No problem with that.

File: loader.js (hunk 2 above)
Comment (robertohuertasm) [reply loader.js]: changed!

File: loader.js (hunk 1 above)
Comment (robertohuertasm) [reply loader.js]: I've kept the format you got there and remove the trailing commas.

File: package.json [hunk 3]
```diff
@@ -24,5 +24,8 @@
   "bugs": {
     "url": "https://github.com/wasm-tool/rust-loader/issues"
   },
-  "homepage": "https://github.com/wasm-tool/rust-loader#readme"
+  "homepage": "https://github.com/wasm-tool/rust-loader#readme",
+  "devDependencies": {
+    "console-emoji": "0.0.2"
```
Comment (xtuc) [inline package.json]: I have to say that this dependency scares me a bit (https://www.npmjs.com/package/console-emoji).

File: loader.js [hunk 4]
```diff
@@ -54,19 +53,25 @@ module.exports = function() {
   const cwd = dirname(this.resourcePath);
   const pkgDir = join(cwd, './pkg');
 
+  log(`:information_source:  Compiling your crate...`, 'green');
```
Comment (xtuc) [inline loader.js]: What if we just inline the emoji in the source?

File: package.json (hunk 3 above)
Comment (robertohuertasm) [reply package.json]: Don't worry. I'll remove it.

Comment (robertohuertasm) [reply package.json]: done!

File: loader.js (hunk 4 above)
Comment (robertohuertasm) [reply loader.js]: done!

**Pull request merged**